js/main.js

```diff
@@ -194,13 +192,16 @@
             cropData.w * scale,
             cropData.h * scale);
 
-        //use ocrad.js to extract text from the canvas
-        var resultText = OCRAD(ctx);
-        resultText = resultText.trim();
+        //Replaced OCRAD with Tesseract
+        Tesseract.recognize(ctx).then(function(result1){
+            var resultText = result1.text;
+
+            resultText = resultText.trim();
 
         //show the result
         $('blockquote p').html('&bdquo;' + resultText + '&ldquo;');
         $('blockquote footer').text('(' + resultText.length + ' characters)')
```

Comment:
OCD triggered! :) Can you please push those two lines right one level?

Reply:
Hahaha No Problem.